[FuSa] Add SafetyContextRelationship and SafetyIntegrityLevelType (#1436)
spdx/spdx-3-model: model/FunctionalSafety/Classes/SafetyContextRelationship.md

SPDX-License-Identifier: Community-Spec-1.0

# SafetyContextRelationship

## Summary

Provides functional safety context for an existing lifecycle-scoped relationship.

## Description

SafetyContextRelationship provides functional safety context for a relationship
between SPDX elements. It enables an existing relationship, such as a dependency

> or requirement relationship, to carry a safety integrity level without adding a

Reviewer: ```suggestion
or requirement relationship, to carry a safety integrity level leveraging the existing
```

> new relationship type for each safety-related use case.

Reviewer: ```suggestion
relationship types as needed for safety-related use case.
```
